FILES.deploy-tools/usr/local/lib/ndslabs/ansible/roles/k8s-local-kubectl/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: pull the kubeconfig from the master
+  fetch: src="{{ item }}" dest=/root/SAVED_AND_SENSITIVE_VOLUME/cluster-info/{{ logical_cluster_name }}
+  with_items:
+    - /etc/hosts 
+    - /etc/kubernetes/kubectl.kubeconfig   
+    - /etc/kubernetes/config
+    - /etc/kubernetes/apiserver
+
+#
+# This relies on the clusters added to /etc/hosts from local-etc-hosts-cluster-include
+#
+- name: set kubectl command
+  set_fact: 
+    kubectl: "kubectl --kubeconfig=/root/SAVED_AND_SENSITIVE_VOLUME/cluster-info/{{ logical_cluster_name}}/{{ inventory_hostname }}/etc/kubernetes/kubectl.kubeconfig"
```

Comment:
While this appears to be working while running ansible, the kubeconfig setting doesn't stick around.  If I run kubectl in the deploy-tools container, I get "The connection to the server localhost:8080 was refused" which means that I have to manually configure the kubeconfig each time.  Also, I would still like kubectl to be on the master node.